.travis.yml

```diff
@@ -16,3 +17,11 @@ addons:
     - libenchant-dev
 git:
   depth: false
+deploy:
+  provider: pypi
+  user: Julian
```

Comment:
I'd recommend adding a deploy specific username/password here with maintainer-ship access only to `jsonschema`

eg `julian-jsonschema-deploy`